docker/salt/tethyscore.sls

```diff
@@ -14,6 +14,11 @@
 {% set POSTGRES_PASSWORD = salt['environ.get']('POSTGRES_PASSWORD') %}
 {% set TETHYS_DB_PORT = salt['environ.get']('TETHYS_DB_PORT') %}
 {% set TETHYS_DB_USERNAME = salt['environ.get']('TETHYS_DB_USERNAME') %}
+{% if salt['environ.get']('SKIP_DB_SETUP') %}
```

Comment:
Rather than having this logic here, lets just add the `SKIP_DB_SETUP` environment variable to the `Dockerfile`

Reply:
removed the checking in the tethyscore.sls and added a new env variable into the Dockerfile
https://github.com/tethysplatform/tethys/pull/1013/commits/d99c5cb08cafdd4fba79414880a7c87fa3b73866